src/Template/Pages/home.ctp

```diff
@@ -163,28 +163,52 @@ $cakeDescription = 'CakePHP: the rapid development PHP framework';
                     Our primary goal is to provide a structured framework that enables PHP users at all levels to rapidly develop robust web applications, without any loss to flexibility.
                 </p>
                 <ul>
-                    <li><a href="http://cakefoundation.org/">Cake Software Foundation</a>
-                    <ul><li>Promoting development related to CakePHP</li></ul></li>
-                    <li><a href="http://www.cakephp.org">CakePHP</a>
-                    <ul><li>The Rapid Development Framework</li></ul></li>
-                    <li><a href="http://book.cakephp.org/3.0/en/">CakePHP Documentation</a>
-                    <ul><li>Your Rapid Development Cookbook</li></ul></li>
+                       
+                     <h3>Help and Bug Reports</h3>
```

Comment:
What would be the correct way to do it? What tag should i have used? Do you
suggest i close each <ul> before each <h3> ?

On 9 March 2016 at 23:58, Mark Story <notifications@github.com> wrote:

> In src/Template/Pages/home.ctp
> <https://github.com/cakephp/app/pull/351#discussion_r55605924>:
>
> > @@ -163,28 +163,52 @@ $cakeDescription = 'CakePHP: the rapid development PHP framework';
> >                      Our primary goal is to provide a structured framework that enables PHP users at all levels to rapidly develop robust web applications, without any loss to flexibility.
> >                  </p>
> >                  <ul>
> > -                    <li><a href="http://cakefoundation.org/">Cake Software Foundation</a>
> > -                    <ul><li>Promoting development related to CakePHP</li></ul></li>
> > -                    <li><a href="http://www.cakephp.org">CakePHP</a>
> > -                    <ul><li>The Rapid Development Framework</li></ul></li>
> > -                    <li><a href="http://book.cakephp.org/3.0/en/">CakePHP Documentation</a>
> > -                    <ul><li>Your Rapid Development Cookbook</li></ul></li>
> > +
> > +                     <h3>Help and Bug Reports</h3>
>
> This isn't valid HTML. h3 cannot be a direct child of ul.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cakephp/app/pull/351/files#r55605924>.
>